randovania/games/samus_returns/logic_database/Area 3 - Factory Exterior.txt

```diff
@@ -498,10 +516,9 @@ Extra - asset_id: collision_camera_035
       All of the following:
           Morph Ball
           Any of the following:
-              Power Bomb or Plasma Beam or Spider Ball or Shoot Beam Burst
-              All of the following:
-                  Damage Boost (Beginner) and Normal Damage ≥ 30
-                  Phase Drift or Normal Damage ≥ 40
+              # Get around Autrack
+              Plasma Beam or Lay Power Bomb or Shoot Beam Burst or Use Spider Ball
```

Comment:
Adjust grouping so DB looks cleaner (pretty print is fine) (move spider into the or with plasma etc)
![image](https://github.com/randovania/randovania/assets/38679103/eebef63d-e312-441a-8bcf-82a7fe5b93a2)